docs/topics/api/activity.rst

```diff
@@ -39,17 +39,22 @@ Review Notes Detail
 
 This endpoint allows you to fetch a single review note for a specific version of an add-on.
 
+    .. note::
+        To allow reviewers to stay anonymous if they wish, the ``user`` object ``name`` can point to
+        their "reviewer" name depending on the action. In addition all other fields in that object,
+        despite being present for backwards-compatibility, are set to ``null``.
```

Comment:
To keep things simple I've done that for all actions, since [the request in the issue](https://github.com/mozilla/addons-server/issues/11002#issuecomment-476845441) was to remove the links to user profiles entirely anyway.

I should remove all that in v4 yeah - I did the simplest thing that was still compatible with v3.

Reply:
10bc305de5fe220fa4cf7ed75aabeb5070049026